ext/mbstring/mbstring.c

```diff
@@ -2509,6 +2510,16 @@ PHP_FUNCTION(mb_convert_encoding)
 	const mbfl_encoding *to_encoding = php_mb_get_encoding(to_encoding_name, 2);
 	if (!to_encoding) {
 		RETURN_THROWS();
+	} else if (to_encoding->no_encoding <= mbfl_no_encoding_charset_min) {
```

Comment:
This also deprecates the `8bit` encoding, which is actually heavily used. It's used to make mbstring work with binary data (this used to be more important when func_overload existed). I don't think there's any reason to deprecate 8bit. Not really sure about 7bit. That one seems less useful, but I think it's also fairly harmless.